packages/jest-snapshot/src/SnapshotFile.js

```diff
@@ -43,11 +45,22 @@ class SnapshotFile {
   }
 
   save() {
+    const operationResult = {};
+    this._uncheckedKeys.forEach(key => delete this._content[key]);
     const serialized = JSON.stringify(this._content);
+
     if (serialized !== '{}') {
       ensureDirectoryExists(this._filename);
       fs.writeFileSync(this._filename, serialized);
+      operationResult.saved = true;
```

Comment:
`return {saved: true, deleted: false}` here and the opposite below. There is no need for an object that has different fields that the engine cannot optimize for. Every API should have consistent return values :)